src/Database/Schema/TableSchema.php

```diff
@@ -140,6 +140,10 @@ class TableSchema implements TableSchemaInterface, SqlGeneratorInterface
             'fixed' => null,
             'collate' => null,
         ],
+        'char' => [
+            'collate' => null,
+            'fixed' => true,
```

Comment:
This should be removed now?  And `'fixed' => null` above too?

Reply:
Yes for both char and string.